circle.yml

```diff
@@ -19,7 +19,7 @@ dependencies:
 
 test:
   override:
-    - ./gradlew clean :core:install :core:verify :mapzen-android-sdk:verify :mapzen-places-api:verify --configure-on-demand
+    - ./gradlew clean :core:install :core:verify :mapzen-android-sdk:verify :mapzen-places-api:verify --configure-on-demand -PdisablePreDex
```

Comment:
The only information I found for this is its recommended to use `-PdisablePreDex` when doing a one time build (typical with CI env.). But no finer details behind the reasoning. Can you point me to a reference/explain? Thanks